Pepperdash Core/Pepperdash Core/Web/RequestHandlers/DefaultRequestRequestHandler.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Crestron.SimplSharp.WebScripting;
+
+namespace PepperDash.Core.Web.RequestHandlers
+{
+	/// <summary>
+	/// Web API default request handler
+	/// </summary>
+	public class DefaultRequestRequestHandler : WebApiBaseRequestHandler

```

Comment:
This class can be simplified by removing the overrides, as they are the same as the methods in the base class. Virtual methods only need to be overridden when something other than the functionality provided by the base class is required.

Reply:
Updated PepperDashCore and removed overrides as suggested.